frontend/web/components/CompareIdentities.tsx

```diff
@@ -120,6 +127,39 @@ const CompareIdentities: FC<CompareIdentitiesType> = ({
     )
   }
 
+  const clonedentityValues = (
+    leftIdentityName: string,
+    rightIdentityName: string,
+    leftIdentityId: string,
+    rightIdentityId: string,
+    environmentId: string,
+  ) => {
+    return openConfirm({
+      body: (
+        <div>
+          {'Are you sure you want to clone all the feature states from '}
```

Comment:
Does cloning clone overrides and traits?

If so I think that might read better

"Cloning an identity will copy any traits and identity overrides. Are you sure?"

Reply:
We don't clone the traits.
Maybe could be: "Cloning {identityA} will copy any identity overrides in {identityB}. Are you sure?"